app/code/Magento/SalesGraphQl/etc/schema.graphqls

```diff
@@ -2,7 +2,8 @@
 # See COPYING.txt for license details.
 
 type Query {
-    customerOrders: CustomerOrders @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\Orders") @doc(description: "List of customer orders")
+    customerOrders: CustomerOrders @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\Orders") @doc(description: "List of customer orders"),
+    currentSessionLastOrder: CustomerOrder @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\CurrentSessionLastOrder") @doc(description: "List of customer orders"),
```

Comment:
Please change description of method

Reply:
Thanks for the feedback and I have just done this.